Comp2024/src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -150,7 +150,10 @@ public Intake( )
     m_ccValid =
         PhoenixUtil6.getInstance( ).canCoderInitialize6(m_CANCoder, "Intake Rotary", CTREConfigs6.intakeRotaryCancoderConfig( ));
     if (Robot.isReal( ))
-      m_rotaryMotor.setPosition(Conversions.degreesToInputRotations(m_currentDegrees, kRotaryGearRatio));
+    {
```

Comment:
Don't quite understand the need for this change, because reading the rotaryMotor when we start up is NOT guaranteed to be anywhere. Let this be done in the periodic method.